gdal.yaml

```diff
@@ -66,37 +76,77 @@ pipeline:
   - runs: |
       rm -rf /usr/lib/cmake/zstd
 
-  - uses: cmake/configure
-
-  - uses: cmake/build
-
-  - uses: cmake/install
-
-  - uses: strip
-
 subpackages:
-  - name: gdal-dev
+  - range: py-versions
+    name: gdal-py${{range.key}}
+    description: ${{package.name}} for Python ${{range.key}}
+    dependencies:
+      provides:
+        - gdal
+      provider-priority: ${{range.value}}
+    pipeline:
+      - uses: cmake/configure
+        with:
+          opts: |
+            -DPython_LOOKUP_VERSION=${{range.key}} \
+            -DPython_ROOT=/usr/lib/python${{range.key}}
+      - uses: cmake/build
+      - uses: cmake/install
+      - uses: strip
+      - runs: |
+          cmake \
+            --build output \
+            --target clean
+
+  - range: py-versions
+    name: gdal-py${{range.key}}-dev
+    description: ${{package.name}}-dev for Python ${{range.key}}
+    dependencies:
+      provides:
+        - gdal-dev
+      provider-priority: ${{range.value}}
     pipeline:
       - uses: split/dev
+        with:
+          package: gdal-py${{range.key}}
     test:
       pipeline:
-        - runs: |
-            gdal-config --version
+        - runs: gdal-config --version
         - uses: test/pkgconf
 
-  - name: py3-gdal
+  - range: py-versions
+    name: py${{range.key}}-gdal
+    description: ${{package.name}} installed for Python ${{range.key}}
+    dependencies:
+      provides:
+        - py3-gdal
+      provider-priority: ${{range.value}}
     pipeline:
       - runs: |
-          mkdir -p ${{targets.subpkgdir}}/usr/lib/
-          mv ${{targets.destdir}}/usr/lib/python3* ${{targets.subpkgdir}}/usr/lib/
+          mkdir -p ${{targets.contextdir}}/usr/lib/
+          mv ${{targets.outdir}}/gdal-py${{range.key}}/usr/lib/python3* ${{targets.contextdir}}/usr/lib/
```

Comment:
`targets.outdir` is not defined, and our linter didn't catch it. I'm trying to fix that in https://github.com/wolfi-dev/os/pull/32135 but it's failing because that path doesn't exist under `targets.destdir`. Do you happen to see what the error is?